spynnaker8/models/synapse_dynamics/synapse_dynamics_structural.py

```diff
@@ -0,0 +1,69 @@
+from spynnaker.pyNN.models.neuron.synapse_dynamics \
+    import SynapseDynamicsStructural as _BaseClass
```

Comment:
why the rename? it obscures the readability, remove the rename and keep as SynapseDynamicsStructural

Reply:
See https://github.com/SpiNNakerManchester/sPyNNaker8/blob/11f2fe1182f744c9d4cfff0299c95e3e97c53359/spynnaker8/models/synapse_dynamics/synapse_dynamics_static.py#L1-L2

Reply:
thats my point. i had to go up there. its not easy readability, and is not any context, youve actively made the code less readable with that change